### PR TITLE
[FIX] website: disable anchor option for popup

### PR DESCRIPTION
**Pull request by bvr-odoo**
task-2172312

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/snippets.xml [hunk 1]
```diff
@@ -935,7 +935,7 @@
     <!-- Anchor Name -->
     <div data-js="anchor"
         data-selector=":not(p).oe_structure > *, :not(p)[data-oe-type=html] > *"
-        data-exclude=".modal *, .oe_structure .oe_structure *, [data-oe-type=html] .oe_structure *">
+        data-exclude=".modal *, .oe_structure .oe_structure *, [data-oe-type=html] .oe_structure *, .s_popup, .o_newsletter_popup">
```
Comment (qsm-odoo) [inline addons/website/views/snippets/snippets.xml:938]: `o_newsletter_popup` is not a class of the website application

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Comment by bvr-odoo** [comment]
> @qsm-odoo, you may want to rebuild or fix this PR as it has failed CI.

@qsm-odoo I have rebuilt and now it works

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**